docker-vertica-v2/Makefile

```diff
@@ -1,14 +1,18 @@
-VERTICA_RPM?=$(notdir $(wildcard packages/vertica*.rpm))
+VERTICA_X86_RPM?=$(notdir $(wildcard packages/vertica*.x86_64.rpm))
+VERTICA_ARM64_RPM?=$(notdir $(wildcard packages/vertica*.aarch64.rpm))
 BUILDER_OS_NAME?=almalinux
 BUILDER_OS_VERSION?=8
 BASE_OS_NAME?=rockylinux
 BASE_OS_VERSION?=9
 FOR_GITHUB_CI?=false
 VERTICA_IMG?=vertica-k8s
 MINIMAL_VERTICA_IMG?=
-VERTICA_VERSION?=$(shell rpm --nosignature -qp --queryformat '%{VERSION}-%{RELEASE}' packages/$(VERTICA_RPM))
+TARGET_ARCH?=linux/amd64
 VERTICA_ADDITIONAL_DOCKER_BUILD_OPTIONS?=
 
+VERTICA_X86_VERSION?=$(shell rpm --nosignature -qp --queryformat '%{VERSION}-%{RELEASE}' packages/$(VERTICA_X86_RPM))
+VERTICA_ARM_VERSION?=$(shell rpm --nosignature -qp --queryformat '%{VERSION}-%{RELEASE}' packages/$(VERTICA_ARM64_RPM))
```

Comment:
These two variables are not used in docker-build-vertica. We should assign the value of them to VERTICA_VERSION.

Reply:
Right, I had initally planned of using this to catch version mismatch if someone does not use the same RPM version. Do you think that is useful and a valid concern?

Reply:
Is this for the case "TARGETARCH == linux/arm64,linux/amd64"? Yes, we need to verify both of them have the same version. And we need to label the version to the image.